src/dma.rs

```diff
@@ -341,14 +341,14 @@ macro_rules! impl_channel {
                     handle:  &mut Handle,
                     address: u32,
                 ) {
-                    handle.dma.$chfield.par.write(|w| w.pa().bits(address));
+                    handle.dma.$chfield.par.write(|w| unsafe{w.pa().bits(address)});
```

Comment:
I suspect this (and the corresponding line below) is perfectly fine, but I'd prefer if you double-checked against the reference documentation, to see which values are actually allowed in this field, and add a comment to explain why this use of `unsafe` is sound (e.g. "This is sound, as PA accepts all possible values of `u32`"; don't know if that's the case, just giving you an example of the kind of comment I'm looking for).